Automattic/wp-calypso
# Add new migration instructions feature flag

Related to #91773

## Proposed Changes

* It adds feature flag for the new migration instructions step.
* It also adds a new component where we can develop the new step.

## Why are these changes being made?


* We're preparing the codebase to start the development of the new migration instructions step.

## Testing Instructions



* Make sure to run `yarn start` again to the new feature flags have effect.
* Test the migration flow in the 3 different flows and make sure it's loading a new component instead of the current migration instructions.
  * `/start`
  * `/setup/migration-signup`
  * `/setup/hosted-site-migration?from={FROM_URL}` (It starts in https://wordpress.com/move)
* Now update the feature flag to `false` also in development, and run the `yarn start` again.
* Run the flows again, and make sure you see the original behavior.

## Pre-merge Checklist
